tests/ut/utils.h

```diff
@@ -46,6 +46,17 @@ GenDataSet(int rows, int dim, int seed = 42) {
     return ds;
 }
 
+inline knowhere::DataSetPtr
+GenFloatDataSet(int rows, int dim, int seed = 42) {
```

Comment:
Where does this link to?

Reply:
ut; previously all generated data are integer